### PR TITLE
Save severity in notice.Context[severity].

### PR DESCRIPTION
**Pull request by anmic**

### CODE REVIEW

**Comment by vmihailenco** [comment]
Did you try to send an error using this version of the notifier?

**Comment by vmihailenco** [comment]
If so please post a link to the notice at airbrake.io.

**Comment by anmic** [comment]
https://getexceptional.err.io/projects/99798/groups/1860986075304960261?tab=notice-detail

File: glog.go [hunk 1]
```diff
@@ -33,17 +33,19 @@ type severity int32 // sync/atomic int32
 // A message written to a high-severity log file is also written to each
 // lower-severity log file.
 const (
-	infoLog severity = iota
+	invalidSeverity severity = iota
+	infoLog
```
Comment (vmihailenco) [inline glog.go]: revert this

File: glog.go [hunk 2]
```diff
@@ -33,17 +33,19 @@ type severity int32 // sync/atomic int32
...
 var severityName = []string{
-	infoLog:    "INFO",
-	warningLog: "WARNING",
-	errorLog:   "ERROR",
-	fatalLog:   "FATAL",
+	invalidSeverity: "NIL",
+	infoLog:         "INFO",
+	warningLog:      "WARNING",
+	errorLog:        "ERROR",
+	fatalLog:        "FATAL",
```
Comment (vmihailenco) [inline glog.go]: and revert this too

**Pull request merged**